components/UnifiedDocFeed/UnifiedDocFeedContainer.tsx

```diff
@@ -75,6 +75,10 @@ function UnifiedDocFeedContainer({
     hotV2: router.query?.hot_v2 == "true",
   };
 
+  useEffect(() => {
```

Comment:
can we remove this state and set it back to what we had before with the useState(false), or do we need it?

Reply:
We need it to avoid a strange bug with the first 3 cards. Not sure why this happens.

<img width="1357" alt="Screen Shot 2022-04-07 at 7 55 54 PM" src="https://user-images.githubusercontent.com/22990196/162347908-1f40c6c2-c07f-44ec-a718-748f1ecfa066.png">